content/swift/concepts/optionals/optionals.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: 'Optionals'
+Description: 'Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Optionals'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Optionals are types that deal with scenarios involving the potential presence or absence of a value (`nil`).
+
+## Syntax
+
+In Swift, a non-optional variable should not be set to `nil`. Rather, the optional type must be defined using the added question mark `?`.
+
+There are two ways to use an optional. Below, a variable named `myVariable` is set to an optional `Type`, `Type?`. `Type` referred to a [Swift data type](https://www.codecademy.com/resources/docs/swift/data-types). It can either be equal to some value with the specified data type or `nil`.
+
+```pseudo
+var myVariable: Type?
+var myVariable: Optional<Type>
+```
+
+**Note:** If `myVariable` is not assigned a value of the specified data type, it will default to `nil`.
+
+## Unwrapping Optionals
+
+Optional types should be unwrapped only if it's certain that a value is present. An [error](https://www.codecademy.com/resources/docs/general/error) will result if an optional without a value is unwrapped. Below are some ways this can be done.
+
+### Nil-Coalescing
+
+The nil-coalescing operator, `??`, assigns a value by checking from left to right. If the preceding optional values are `nil`, the default value on the right is assigned. In the example below, since `maybeInt` was previously declared as `nil`, `myInt` will default to `27`:
```

Comment:
should there be a line of code showing the definition of `maybeInt`. I'm assuming it looks something  like this:
```swift
var maybeInt:Int? = nil
```
Since Swift is fussy when a type is not declared. Should the `:Int?` part be explained?